core/server/worker/src/test/java/alluxio/worker/block/stream/BlockWorkerDataReaderTest.java

```diff
@@ -122,7 +122,7 @@ public void before() throws Exception {
     URIStatus dummyStatus =
         new URIStatus(new FileInfo().setBlockIds(Collections.singletonList(BLOCK_ID)));
     InStreamOptions options =
-        new InStreamOptions(dummyStatus, FileSystemOptions.openFileDefaults(mConf), mConf);
+        new InStreamOptions(dummyStatus, FileSystemOptions.openFileDefaults(mConf), mConf, null);
```

Comment:
Actually, if you create a `FileSystemContext` here can we already be sure that in the constructor the `FileSystemContext` is never null? Then we can just do `Precondition.notNull(fsContext)` in that constructor?